modules/browser/BrowserCapabilities.js

```diff
@@ -241,6 +255,39 @@ export default class BrowserCapabilities extends BrowserDetection {
         return this.isFirefox();
     }
 
+    /**
+     * Checks if the browser uses unified plan by default.
+     *
+     * @returns {boolean}
+     */
+    isUnifiedPlanDefault() {
+        if (this.isFirefox()) {
+            return true;
+        }
+
+        if (this.supportsSdpSemantics()) {
```

Comment:
This branch is not currently run, and the rought plan moving forward is to set SDP semantics to Plan B until we fully migrate to Unified, so I don't think we'll ever need this check for Chrome.

Reply:
I'll remove that. I've just wanted to make a generic function.

Reply:
Let's drop this now, since Safari won't implement it and it's Chrome only but we already pass the plan-b constraint.